### PR TITLE
Strapi

### PR DESCRIPTION
**Pull request by gittysachin**
Adding strapi integration https://github.com/pluralsh/plural-artifacts/issues/132. Initial push for testing

### CODE REVIEW

File: strapi/helm/strapi/templates/tests/test-connection.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```
Comment (michaeljguarino) [inline strapi/helm/strapi/templates/tests/test-connection.yaml:1]: Usually it's a good idea to drop these helm tests as they can make deployments more flaky

File: strapi/helm/strapi/values.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# Default values for strapi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: gittysachin/strapi
```
Comment (michaeljguarino) [inline strapi/helm/strapi/values.yaml:8]: you should be able to reference this as `dkr.plural.sh/strapi/gittysachin/strapi` once the vendor images action runs.

**Review by michaeljguarino — COMMENTED** [review]
This looks pretty good, some minor suggestions but overall this looks good to ship!

**Comment by michaeljguarino** [comment]
No need to update docs.plural.sh for this btw, we can handle that ourselves (there's a script we've written that does it)

File: strapi/helm/strapi/values.yaml (hunk 2 above)
Comment (gittysachin) [reply strapi/helm/strapi/values.yaml:8]: @michaeljguarino got it, so should I change the repository name to include `dkr.plural.sh/strapi` now?

File: strapi/helm/strapi/templates/tests/test-connection.yaml (hunk 1 above)
Comment (gittysachin) [reply strapi/helm/strapi/templates/tests/test-connection.yaml:1]: Removing

**Review by michaeljguarino — APPROVED** [review]
This looks pretty good, some minor suggestions but overall this looks good to ship!

File: strapi/helm/strapi/values.yaml (hunk 2 above)
Comment (gittysachin) [reply strapi/helm/strapi/values.yaml]: Changed the image url to `dkr.plural.sh`

**Pull request merged**